### PR TITLE
addForm connected with database

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: addComand.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolioProject', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = "INSERT INTO `portfolio` (`project_pic`, `url`) VALUES (:project_pic, :url);";
+
+$projectPic = $_POST['project_pic'];
```
Comment (mporam) [inline addComand.php]: no check if project_pic exists

Comment (mporam) [reply addComand.php]: or url on the line below

File: addComand.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+$url = $_POST['url'];
+
+if (!empty($projectPic) && (!empty($url))) {
```
Comment (mporam) [inline addComand.php]: this if is useless as it will error on the line above if the form was empty

File: addComand.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    $query = $db->prepare($sql);
+    $query->bindParam(':project_pic', $projectPic, PDO::PARAM_STR);
+    $query->bindParam(':url', $url, PDO::PARAM_STR);
+    $query->execute();
+    header('location: adminPanel.php');
```
Comment (mporam) [inline addComand.php]: no check if the query runs correctly. use:
```
$result = $query->execute();
if ($result) {
header('location: adminPanel.php');
} else {
//show error
}
```

File: addComand.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        $url = $_POST['url'];
+        $query = $db->prepare($sql);
+        $query->bindParam(':project_pic', $projectPic, PDO::PARAM_STR);
+        $query->bindParam(':url', $url, PDO::PARAM_STR);
+        $query->execute();
+        $result = $query->execute();
+        if ($result) {
+            header('location: adminPanel.php');
+        } else {
+            echo '<p>Oops! Something went wrong. This page didn\'t load properly</p>';
```
Comment (mporam) [inline addComand.php]: lines 19 & 20 are identical to lines 23 & 23 - please put them into a function

File: addComand.php (hunk 1 above)
Comment (alecorra) [reply addComand.php]: check statement added

File: addComand.php (hunk 2 above)
Comment (alecorra) [reply addComand.php]: both variables moved into statement

File: addComand.php (hunk 3 above)
Comment (alecorra) [reply addComand.php]: done

File: addComand.php (hunk 4 above)
Comment (alecorra) [reply addComand.php]: function created

File: addComand.php [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolioProject', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = "INSERT INTO `portfolio` (`project_pic`, `url`) VALUES (:project_pic, :url);";
+
+function errorMsg(): string {
```
Comment (mporam) [inline addComand.php]: can you take this function out and put it into a functions.php file? then include it into this file. that way you have a central file for all functions (if you add more) and it makes them quicker/easier to test

Comment (alecorra) [reply addComand.php]: done

**Pull request merged**